doc/classes/Image.xml

```diff
@@ -618,7 +618,8 @@
 		<method name="srgb_to_linear">
 			<return type="void" />
 			<description>
-				Converts the raw data from the sRGB colorspace to a linear scale. Only works on images with [constant FORMAT_RGB8] or [constant FORMAT_RGBA8] formats.
+				Converts the raw data from nonlinear sRGB encoding to linear encoding using a lookup table. Only works on images with [constant FORMAT_RGB8] or [constant FORMAT_RGBA8] formats.
+				[b]Note:[/b] The 8 bit formats required by this function are not suitable for storing linearly encoded values because a significant amount of color information will be lost in darker values. To maintain image quality this function should not be used.
```

Comment:
```suggestion
				[b]Note:[/b] The 8-bit formats required by this method are not suitable for storing linearly encoded values, because a significant amount of color information will be lost in darker values. To maintain image quality, this method should not be used.
```